Code/Framework/AzCore/AzCore/Math/MathUtils.h

```diff
@@ -461,6 +461,36 @@ namespace AZ
         return (azisfinite(x) != 0);
     }
 
+    //! Returns the value divided by alignment, where the result is rounded up if the remainder is non-zero.
+    //! Example: alignment: 4
+    //! Value:  0 1 2 3 4 5 6 7 8
+    //! Result: 0 1 1 1 1 2 2 2 2
+    constexpr uint32_t DivideAndRoundUp(uint32_t value, uint32_t alignment)
+    {
+        AZ_Assert(alignment != 0, "0 is an invalid multiple to round to.");
+        return ((value + alignment - 1) / alignment);
+    }
+
+    constexpr uint64_t DivideAndRoundUp(uint64_t value, uint64_t alignment)
+    {
+        AZ_Assert(alignment != 0, "0 is an invalid multiple to round to.");
+        return ((value + alignment - 1) / alignment);
```

Comment:
uber nit: external parens are redundant (if this is a 'when in Rome' think and lots of other functions in this file do it it's fine to leave).